### DISCUSSION TITLE
how to computes the grayscale mean

### DISCUSSION THREAD

**Discussion by TheXiong** [dlemstra/magick-wasm#71] [Q&A]
```
var statistics = img.Statistics();
var max = _factory.Quantum.Max.ToDouble(null);
var mean = (statistics.Composite().Mean / max) - 0.5;
```
how can I translate the above c# code.

**Comment by TheXiong** [comment]
@dlemstra

**Comment by dlemstra** [comment]
The `statistics()` method is not yet available in the api. But I think I can add it before the next release.

**Reply by TheXiong** [reply]
ok. about when you release the next version?

**Reply by dlemstra** [reply]
I dont know, probably after I have time to add this.

**Reply by TheXiong** [reply]
ok, thank you, and can you give me a bundle of the branch master? I need the merge API of MagickImageCollection.

**Reply by dlemstra** [reply]
That will also become available in the next release. Is there anything else extra that you need?

**Reply by TheXiong** [reply]
no extra. thank you.

**Reply by dlemstra** [reply]
The new release has been published.

**Reply by TheXiong** [reply]
thank you very much, you really helped me a lot.
